src/pages/index.js

```diff
@@ -57,37 +60,82 @@ const previewCloseButton = previewModal.querySelector(".modal__close-btn");
 
 const previewCaption = previewModal.querySelector(".modal__caption");
 
+const deleteModal = document.querySelector("#delete-modal");
+
+const deleteForm = document.forms[2];
```

Comment:
`document.forms[2];`  

If the order of the forms change in `index.html`, your code will not work anymore because `[2]` will return another form


You should take the forms, using their `name` attribute value


```js
const deleteForm = document.forms["delete-form"];


```